README.md

```diff
@@ -52,15 +52,39 @@ We provide several notebooks to show how recommendation algorithms can be design
 
 - The [Operationalizion Notebook](notebooks/05_operationalize) demonstrates how to deploy models in production systems.
 
+
+The Quick-Start and Modeling notebooks showcase how to utilize the following algorithms to build a recommender system:
+
+**Algorithms**
+
+| Algorithm | Environment | Description | 
+| --- | --- | --- |
+| **`Classic Recommenders`** |
+| [Surprise/Singular Value Decomposition (SVD)](notebooks/00_quick_start/sar_single_node_movielens.ipynb) | Python |  | 
+| [Alternating Least Squares (ALS)](notebooks/00_quick_start/als_pyspark_movielens.ipynb) | Spark |  |
+| **`Microsoft Recommenders`** |
+| [Smart Adaptive Recommendations (SAR)](notebooks/00_quick_start/sar_single_node_movielens.ipynb) | Python / Spark | |
+| [Vowpal Wabbit Family (VW)](notebooks/02_model/vowpal_wabbit_deep_dive.ipynb) | Python / Online | |
+| **`Deep Learning`** |
+| [eXtreme Deep Factorization Machine (xDeepFM)](notebooks/00_quick_start/xdeepfm.ipynb) | Python / GPU | | 
+| [Deep Knowledge-Aware Network (DKN)](notebooks/00_quick_start/dkn.ipynb) | Python / GPU | | 
+| [Neural Collaborative Filtering (NCF)](notebooks/00_quick_start/ncf_movielens.ipynb) | Python / GPU | | 
+| [Restricted Boltzmann Machines (RBM)](notebooks/00_quick_start/rbm_movielens.ipynb) | Python / GPU | | 
+| [FastAI](notebooks/00_quick_start/fastai_recommendation.ipynb)  | Python / GPU | | 
```

Comment:
I think we should rename FastAI to FastAI EmbeddingDotBias https://docs.fast.ai/collab.html#EmbeddingDotBias, which is the algo that we are using. I'm going to make a modification of the fastai notebook to make this more clear